test/index.html

```diff
@@ -11,12 +11,7 @@
 
 <body>
   <script>
-    WCT.loadSuites(
-      window.RichTextEditorSuites
-        .reduce(function(suites, suite) {
-          return suites.concat([suite, `${suite}?wc-shadydom=true`]);
-        }, [])
-    );
+    WCT.loadSuites(window.RichTextEditorSuites);
```

Comment:
Running test suites with ShadyDOM enabled caused issues in Chrome.

Discussed with Serhii and decided that this was not necessary anymore. Browsers without shadow DOM support enable ShadyDOM by default. For other browsers we don't need ShadyDOM enabled, so this is not a valid use-case.